packages/neutrino/bin/start.js

```diff
@@ -1,17 +1,34 @@
 const ora = require('ora');
 const debounce = require('lodash.debounce');
+const DashboardPlugin = require('webpack-dashboard/plugin');
+const { spawn } = require('child_process');
 const { start } = require('../src');
 const base = require('./base');
 
 module.exports = (middleware, args) => {
   const spinner = ora({ text: 'Building project' });
+  const argsClone = Object.assign({}, args);
```

Comment:
Yeah, I think you can avoid the `argsClone` altogether here with either:

```js
if (args.dashboard) {
  Object.assign(args, { quiet: true });
}
```

or:

```js
if (args.dashboard) {
  args.quiet = true; // eslint-disable-line no-param-reassign
};
```

Reply:
Sure, I agree.